cicada/commands/exec_schedule.py

```diff
@@ -242,20 +258,25 @@ def main(schedule_id, dbname=None):
                 zombie.send_signal(signal.SIGTERM)
 
             # Repeatedly attempt to finalize schedule, even if db is unavailable
-            db_connection_made = False
-            while not db_connection_made:
+            db_conn_success = False
+            while not db_conn_success:
                 try:
                     db_conn = postgres.db_cicada(dbname)
                     db_cur = db_conn.cursor()
-                    db_connection_made = True
+                    db_conn_success = True
                 except Exception:
-                    send_slack_error(
-                        schedule_id,
-                        schedule_log_id,
-                        returncode,
-                        "Cicada database not reachable to finalize schedule",
-                    )
-                    time.sleep(1)
+                    if datetime.datetime.utcnow() >= db_conn_alert_next:
+                        send_slack_error(
```

Comment:
Don't you think if it can be helpful to send exception message as well?

Reply:
Might be ... will add it